drivers/apds9007/apds9007.c

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (C) 2018 UC Berkeley
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ *
+ */
+
+/**
+ * @ingroup     drivers_apds9007
+ * @{
+ *
+ * @file
+ * @brief       Driver for the APDS9007 Light Sensor.
+ *
+ * @author      Hyung-Sin Kim <hs.kim@cs.berkeley.edu>
+ *
+ * @}
+ */
+
+#include <string.h>
+
+#include "apds9007.h"
+#include "xtimer.h"
+
+#define ENABLE_DEBUG    (0)
+#include "debug.h"
+
+
+void apds9007_set_active(const apds9007_t *dev)
+{
+    gpio_write(dev->p.gpio, 0);
+}
+
+void apds9007_set_idle(const apds9007_t *dev)
+{
+    gpio_write(dev->p.gpio, 1);
+}
+
+void apds9007_init(apds9007_t *dev, const apds9007_params_t *params)
+{
+    dev->p.gpio = params->gpio;
+    dev->p.adcline = params->adcline;
+    dev->p.adcres  = params->adcres;
+
+    gpio_init(params->gpio, GPIO_OUT);
```

Comment:
```suggestion
    if (params->gpio != GPIO_UNDEF) {
        gpio_init(params->gpio, GPIO_OUT);
    }
```

The sensor will still work if the shutdown pin is not connected / connected to ground and some platforms will crash if you try to initialize / access `GPIO_UNDEF`.